snare.py

```diff
@@ -214,6 +221,13 @@ def snare_setup():
         return snare_uuid
 
 
+def create_initial_config():
+    config = configparser.ConfigParser()
```

Comment:
I think you can move this into the module scope (after the `if __name__ == '__main__'` and parsing the `--config` argument) so we can use it anywhere.